README.md

```
@@ -99,91 +99,83 @@ pylay generate dependency-graph --input src/ --output docs/dependency_graph.png
 ```
 
 `# type: ignore` が使用されている箇所の原因を自動的に特定し、具体的な解決策を提案します：
 - **優先度判定**: HIGH/MEDIUM/LOW で問題を分類
 - **原因特定**: mypy実行による型エラー情報の取得と紐付け
-- **解決策提案**: コンテキストに応じた具体的な修正方法を提示
+- **解決策提案**: コンテキストに応じた具体的な修正方法を提示（`-v`オプションで表示）
 - **モダンUI**: Richベースの見やすいコンソール出力
 
 詳細は [type: ignore診断機能](docs/features/diagnose-type-ignore.md) を参照してください。
 
+### 品質チェック
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)